applications/FluidDynamicsApplication/custom_elements/qs_vms_dem_coupled.cpp

```diff
@@ -481,6 +480,49 @@ void QSVMSDEMCoupled<TElementData>::CalculateTau(
     TauTwo = viscosity + c2 * density * velocity_norm * h / c1;
 }
 
+template< class TElementData >
+void QSVMSDEMCoupled<TElementData>::SubscaleVelocity(
+    const TElementData& rData,
+    array_1d<double,3> &rVelocitySubscale) const
+{
+    BoundedMatrix<double,Dim,Dim> tau_one = ZeroMatrix(Dim, Dim);
+    double tau_two;
+    array_1d<double,3> convective_velocity = this->GetAtCoordinate(rData.Velocity,rData.N) - this->GetAtCoordinate(rData.MeshVelocity,rData.N);
+    this->CalculateTau(rData,convective_velocity,tau_one,tau_two);
+
+    array_1d<double,3> Residual = ZeroVector(3);
+
+    if (rData.UseOSS != 1.0)
```

Comment:
```suggestion
    if (!rData.UseOSS)
```
Note that `UseOSS` is of `int` type.